core/dbt/clients/jinja.py

```diff
@@ -496,7 +496,7 @@ def get_environment(
     args['extensions'].append(TestExtension)
 
     env_cls: Type[jinja2.Environment]
-    text_filter: Type
+    filters: Type
```

Comment:
Honestly, this one is a bit of a guess. Only real change in this PR I'm a bit halfway on.

Reply:
I was wrong and fixed it.